bin/auto-cpufreq

```diff
@@ -238,19 +251,27 @@ def main(config, daemon, debug, update, install, remove, live, log, monitor, sta
                 verify_update()
                 ans = input ("Do you want to update auto-cpufreq to the latest release? [y/n]: ")
                 valid_options = ['y', 'Y', 'yes', 'YES', 'Yes']
+                if custom_dir != "/opt/auto-cpufreq/source":
+                    if not os.path.exists(custom_dir):
+                        input("Directory does not exist, press Enter to create and proceed or ctrl+c to exit")
+                        os.makedirs(custom_dir)
+                else:
```

Comment:
Is logic from line 258 to 261 to delete the dir and its contents, so new contents can be cloned?